saleor/order/models.py

```diff
@@ -289,6 +289,8 @@ class Order(ModelWithMetadata):
     redirect_url = models.URLField(blank=True, null=True)
     search_document = models.TextField(blank=True, default="")
     search_vector = SearchVectorField(blank=True, null=True)
+    # this field is used only for draft/unconfirmed orders
+    should_refresh_prices = models.BooleanField(default=True)
```

Comment:
Question: Based on the comment above. In that case shouldn't we create a  @property with this name, that will take into account the value of refresh_price and current status of order? This could be a "protection" from getting the incorrect value of `should_refresh_prices`